src/reaktoro_pse/tutorials/softening_acid_ro_example.ipynb

```diff
@@ -0,0 +1,1324 @@
p07l+UyERG5O3zeGqNqc9oRomx08/f29pZWrVrJpEmTJCoqSjZt2iRNmjSRTz/9VIoXLy6tW7eW999/X06fPp2p/cbHx0vdunVl6tSpdtePGjVKFi1aJLNnz5b9+/drcIaA6bfffjNvM3LkSPn999/lp59+klWrVsmZM2fu2POOiMjTGaNqIw+J3f2JRLxMDn4nnD9/XgOUBQsWaBD1wgsvZK1gXl4yb9486d69u3lZrVq15JFHHpFx48aZlzVs2FCb/N588025cuWKFC1aVObOnSsPPvigrj9w4IBUr15dNmzYIE2bNs3QY8fFxWlvPOyvYEH26CAi93c9KVnqjV8iSSmpsnRUG6kUXsDZRSLKNEdevzPdi82yNie9wCYgIEADm27duokjNW/eXGuLnnzySW2GW7lypRw8eFCmTJmi65E8fvPmTWnfvr35f6pVqyZlypS5bYCUmJioN8sTTETkSYL8fOWu8qGy9vAFbWZjgESeLssB0vbt22Xbtm2SkpIiVatW1WUIVnx8fDQomTZtmtYerVmzRnOFHOF///ufDB48WHOQfH19tanv888/1yY9QG6Un5+fhISEWP1fsWLFdF16Jk6cKOPHj3dIGYmIXBXykBAgrYyKkQEtyzu7OESuOdUIaodQU4McH9Tc4IYebh06dJBHH31U84/QxIacIEdBgLRx40atRcLjffDBB5qQvXTp0mztd8yYMVodZ9xOnjzpsDITEblaHtKmY5fkRlKKs4tD5Jo1SO+9954sWbLEqo0P7X6vv/66dOzYUZ577jkdMwn3HeHGjRvy3//+V/OSunbtqsvq1KkjO3bs0IRwBGsRERE65ACmO7GsRUIvNqxLj7+/v96IiDwZmtVKhgTK6dgbsvHoRbm72q3xkYg8UZZrkFDTEhMTYzdJ28jhQZDiqDGSkFuEG5rVLKFJLzU11ZywnS9fPlm2bJl5PXrZnThxQpo1a+aQchARuSvkkLb+txaJ3f3J0/lmp4kNydJo5mrcuLEu27Jli+YdGT3PNm/eLFWqVMnwPjHO0eHDh81/Hzt2TGuIQkNDNdG6TZs2Mnr0aB0DqWzZstqNH2MvTZ482VyDNWDAAE0gx/+gdmv48OEaHGW0BxsRkac3s323+YTmIYnUdHZxiJzHlEVXr141DRw40OTn52fy9vbWG+4PGjTIdO3aNd1m+/btesuoFStWYMiBNLd+/frp+rNnz5r69+9vKlGihCkgIMBUtWpV0wcffGBKTU017+PGjRumZ555xlS4cGFTUFCQqUePHvp/mXHlyhV9XPwmIvIkcTeSTBXH/GEq+9JC07Hztz7LiVyFI6/f2R4HCbU+R48e1fsVKlTQ0atdHcdBIiJP9sinGzRRe/wDNaVf83LOLg6RU67fWc5BMiAgQrI0bu4QHBERebq2/05eyzwk8mRZzkEy5j3btWuXJmsbidKGBx54ILtlIyIiJ+UhvbvogGw4clESbqZIQD4fZxeJyHUCJMyJ9vjjj8uFCxfs9oTAAJJEROR6qhcPlvBgf4m5mihb/rkkrSrf6tlG5Emy3MSG3mEPPfSQnD17VmuPLG8MjoiIXBe+5BqDRq6KYjMbeaYsB0gYfBHd6TGNBxERuWce0krmIZGHynKA9OCDD+pksURE5H5aVioi3l4ih2OuyanL151dHCLXyUH65JNPtIkNk9HWrl1bR7C29OyzzzqifERE5ASFgvJJ/TKFJfL4Ze3N1qdJWWcXicg1AqTvvvtO/v77bwkICNCaJLRZG3CfARIRkWtrW6XorQApigESeZ4sN7G98sorMn78eB2M6Z9//tFpQYybMXAkERG5rjZVbyVqrz9yUZKSrYdyIXJ3WQ6QMAntI488kmbyWCIicg+1ShSSsPx+ci0xWWuSiDxJlqObfv36yQ8//ODY0hARUZ7h7e0lrY3u/uzNRh4myzlIGOto0qRJsnjxYp1mxDZJe/LkyY4oHxEROVHbqkVl3vbTsjIqRl6+t5qzi0OU9wOk3bt3S/369fX+nj17rNZZJmwTEZFrd/fHR/qB6KtyLi5BihUMcHaRiPJ2E9uKFSvSvS1fvjxL+1y9erXcf//9UqJECQ2y5s+fn2ab/fv36zxvmK03f/780rhxYzlx4oTV/HBDhw6VsLAwnTy3V69eOqglERFlXlgBf6lTspDe56ja5EkyFSBZBiIZcfr06UxtHx8fL3Xr1pWpU6faXX/kyBFp2bKlVKtWTYcWwES548aN06EGDCNHjpTff/9dfvrpJ1m1apWcOXNGevbsmalyEBHR/2vz76jazEMiT+JlMplMGd0Y04p0795dBg4cqDU39qDb/48//igfffSRDB48OMvjIaEGad68efp4ht69e2uu07fffpvuYxctWlTmzp2rI33DgQMHpHr16rJhwwZp2rRphh47Li5Oa6iwv4IFC2ap/ERE7gI92HpNXy8FA3xl27gO4uvD3suUNzny+p2pV/m+ffu0WatDhw4SEREhXbt2lUGDBunEtf/5z3+kQYMGEh4eLl999ZUmcDtysEhMgvvHH39IlSpVpFOnTvo4TZo0sWqGi4yMlJs3b0r79u3Ny1DbVKZMGQ2QiIgo8+qVDpFCgfkkLiFZdpyMdXZxiHJFpgIk5PWgd9rZs2d1qpHKlSvLhQsX5NChQ7q+T58+GqQgGOnSpYtDCxoTEyPXrl2Td955Rzp37qyjePfo0UObz9CUBtHR0eLn5ychISFpar6wLj2JiYkadVreiIjoFh9vL2lVuYjeZzMbeYos9WILDAzUJiyjGSs3oAYJunXrpnlGUK9ePVm/fr3MmDFD2rRpk+V9T5w4UUcFJyIi+9pWDZeFu87Kyqjz8nzHqs4uDlGOc5mG5CJFioivr6/UqFHDajnyi4zkcTT7YYTv2FjrKmD0YsO69IwZM0bbK43byZMnc+goiIhcU+sqt2qQdp++IheuJTq7OEQ5zmUCJDSdITE8KirKavnBgwelbNlbkyg2bNhQk7iXL
```

Comment:
Whew! I previously noticed a spike that looked like a numerical issue, but seems you fixed that. Looks good!